File: mobile_app/app/src/main/java/com/example/licenta_mobile/dialog/ExistingReservationDialog.java
package com.example.licenta_mobile.dialog;

import android.content.Context;
import android.content.DialogInterface;

import androidx.appcompat.app.AlertDialog;

public class ExistingReservationDialog {

    public static void show(Context context){
        AlertDialog.Builder builder = new AlertDialog.Builder(context);
        builder.setMessage("There is already a pending reservation for this license plate.\nFinish the existing reservation first in order to be able to make a new one")
                .setCancelable(false)
                .setTitle("Reservation Info")
                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
                    public void onClick(DialogInterface dialog, int id) {

                    }
                });
        AlertDialog alert = builder.create();
        alert.show();
    }
}

File: mobile_app/app/src/main/java/com/example/licenta_mobile/dialog/ExistingReservationDialog.kt
package com.example.licenta_mobile.dialog

import android.content.Context
import androidx.appcompat.app.AlertDialog

object ExistingReservationDialog {

    fun show(context: Context) {
        val builder = AlertDialog.Builder(context)
        builder.setMessage("There is already a pending reservation for this license plate.\nFinish the existing reservation first in order to be able to make a new one.")
                .setCancelable(false)
                .setTitle("Reservation Info")
                .setPositiveButton("OK") { _, _ -> }
        val alert = builder.create()
        alert.show()
    }
}